substrate/frame/revive/fixtures/contracts/Deposit.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.20;
+
+contract Deposit {
+  uint256 a;
+  uint256 b;
+
+  function clearStorageSlot(uint256 slot) internal {
+    address storagePrecompile = 0x0000000000000000000000000000000000000901;
+    bytes memory key = abi.encodePacked(bytes32(slot));
+    (bool _success, ) = storagePrecompile.delegatecall(
+        abi.encodeWithSignature("clearStorage(uint32,bool,bytes)", 0, true, key)
+    );
+  }
+
+  function clear() external {
+    clearStorageSlot(0);
+    clearStorageSlot(1);
+  }
+
+  function c() external {
```

Comment:
nit: would be goot to give these function proper names, it makes things easier to follow from the rust code